firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/CrashlyticsCore.java

```diff
@@ -108,21 +109,22 @@ public CrashlyticsCore(
       BreadcrumbSource breadcrumbSource,
       AnalyticsEventLogger analyticsEventLogger,
       FileStore fileStore,
-      ExecutorService crashHandlerExecutor,
       CrashlyticsAppQualitySessionsSubscriber sessionsSubscriber,
-      RemoteConfigDeferredProxy remoteConfigDeferredProxy) {
+      RemoteConfigDeferredProxy remoteConfigDeferredProxy,
+      CrashlyticsWorker commonWorker,
```

Comment:
I was thinking about pass firebase common background here then init crashlytics work within the core initialization.

Reply:
We use common worker for settings in this class.